cmd/buildah/from.go

```diff
@@ -188,7 +188,7 @@ func fromCmd(c *cli.Context) error {
 		return err
 	}
 
-	transport := buildah.DefaultTransport
+	transport := ""
 	arr := strings.SplitN(args[0], ":", 2)
 	if len(arr) == 2 {
```

Comment:
What is `transport := ""` supposed to do? `newBuilder` sets `options.Transport` to `DefaulTransport` if it is `""`, so this doesn’t change behavior at all AFAICT.

If I understand the code right, `BuilderOptions.Transport` needs to exist basically because if `util.ResolveName` detects the _transport_`:`_name_ syntax, it _throws away the transport_ and returns only the _name_ part, which forces `resolveImage` to use `BuilderOptions.Transport` to reconstruct the original input. Working around that the internal implementation throws data away should not be the responsibility of `cmd/buildah` at all; that data should not be thrown away in the first place, i.e. maybe `util.ResolveName` should return a (source to pull, name to use for local storage) pairs instead of just strings.

… OK, now I see `useRegistries :=` now. That’s entirely in the wrong place, an imprecise heuristic to determine whether we used the search list when the code, at some point, _knows precisely_ whether that was the case, and this seemingly-no-op change exists only to make that heuristic work.